src/views/Playgrounds/PlaygroundValidationForm.tsx

```diff
@@ -117,6 +119,38 @@ const PlaygroundValidationForm = () => {
           />
         </FormField>
 
+        <FormField title="Content Rating" description="Lorem ipsum dolor sit amet.">
+          <Controller
+            name="radioGroup"
+            control={control}
+            rules={{ required: true }}
+            render={(props) => (
+              <StyledRadioContainer>
+                <RadioButton
+                  value="all"
+                  label="All audiences"
+                  onChange={(e) => {
+                    clearErrors('radioGroup')
+                    setValue('radioGroup', e.currentTarget.value)
+                  }}
+                  selected={props.value}
+                  error={!!errors.radioGroup}
+                />
+                <RadioButton
+                  value="mature"
+                  label="Mature"
+                  onChange={(e) => {
+                    clearErrors('radioGroup')
+                    setValue('radioGroup', e.currentTarget.value)
+                  }}
+                  selected={props.value}
+                  error={!!errors.radioGroup}
+                />
```

Comment:
Checking on the playground, not sure if this might be an issue with the radio buttons themselves or more with the useForm hook but if I click on both options a couple times it stops changing the values